### ISSUE TITLE
jar urls does not working anymore

### ISSUE THREAD

**Issue by alexsuter** [classgraph/classgraph#434]
I have upgraded to latest classgraph '4.8.52' > '4.8.83' version. Now, something behaves different.

Urls starting with file are still working:
`file:///folder/test/my.iar!classes/`

Urls with another scheme does not work anymore:
`jar:///folder/test/my.iar!classes/`
`jar:file///folder/test/my.iar!classes/`

I'm using `overrideClasspath`, like:
```
var classGraph = new ClassGraph()
            .addClassLoader(classLoader)
            .overrideClasspath(urls)
            .disableNestedJarScanning()
            .enableAnnotationInfo();
```

I can see in the release notes that may something has changed, but I'm not sure. I'm good with the current solution, because I just can modify the urls and replace these two special schemes with the file-scheme. I just want to make sure, if this change was intended or maybe a bug.

Thanks for your work and this library is awesome!

**Comment by lukehutch** [comment]
Thanks for the report. This is fixed in git master, and will be in the next release.

**Comment by lukehutch** [comment]
Fixed and released in 4.8.84. Thanks for the report!
